helpers/tercerosHelper/tercero.helper.go

```diff
@@ -202,3 +202,29 @@ func GetTerceroByDoc(doc string) (tercero *models.DatosIdentificacion, outputErr
 	}
 
 }
+
+func GetTerceroById(id int) (tercero *models.Tercero, outputError map[string]interface{}) {
```

Comment:
Falta documentar, cambiar `...` en el _suggested change_, digamos "GetTerceroById hace tal cosa"
```suggestion
// GetTerceroById ...
func GetTerceroById(id int) (tercero *models.Tercero, outputError map[string]interface{}) {
```